### DISCUSSION TITLE
Add new model script/table in dbt_artifacts package

### DISCUSSION THREAD

**Discussion by bhratkapoor** [brooklyn-data/dbt_artifacts#211] [Q&A]
Hi Team,

is there any possibility to add new model script/table in existing package.

I tried to create that one, table is created, but that model script is not called when i execute my project mode.

Any help on this...

**Comment by bhratkapoor** [comment]
@iribarnesy, any help on this

**Comment by jaypeedevlin** [comment]
Hi Bhrat, can you explain in more detail what you're trying to do and why? That might help us answer your question.
